src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -79,7 +79,7 @@ export enum MutationTableColumnType {
 
 type MutationTableColumn = Column<Mutation[]>&{order:number};
 
-class MutationTableComponent extends MSKTable<Mutation[]> {
+class MutationTableComponent extends LazyMobXTable<Mutation[]> {
```

Comment:
yes its also going to conflict with https://github.com/cBioPortal/cbioportal-frontend/pull/322